server/models/LibraryItem.js

```diff
@@ -237,35 +237,7 @@ class LibraryItem extends Model {
    * @returns {Promise<boolean>} true if updates were made
    */
   static async fullUpdateFromOld(oldLibraryItem) {
-    const libraryItemExpanded = await this.findByPk(oldLibraryItem.id, {
-      include: [
-        {
-          model: this.sequelize.models.book,
-          include: [
-            {
-              model: this.sequelize.models.author,
-              through: {
-                attributes: []
-              }
-            },
-            {
-              model: this.sequelize.models.series,
-              through: {
-                attributes: ['id', 'sequence']
```

Comment:
Turns out we were using the `bookSeries.id` to handle updating series sequence. https://github.com/advplyr/audiobookshelf/issues/3636
I looked for this specifically and didn't see it. Just an fyi

Reply:
Good catch. I didn't notice the difference.